codec/pom.xml

```diff
@@ -83,6 +84,12 @@
       <artifactId>lzma-java</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.github.luben</groupId>
+      <artifactId>zstd-jni</artifactId>
+      <version>${ztsd-version}</version>
```

Comment:
Please make this extra dependency optional.

Reply:
Thanks for your suggestion

Reply:
I've finished